satellitevu/client.py

```diff
@@ -3,19 +3,21 @@
 
 from satellitevu.apis.archive import ArchiveV1
 from satellitevu.apis.orders import OrdersV1
-from satellitevu.apis.otm import OtmV1
+from satellitevu.apis.otm import OtmV1, OtmV2
 from satellitevu.auth import AbstractCache, Auth
 from satellitevu.config import GATEWAY
 from satellitevu.http import AbstractClient, UrllibClient
 
 
 class FutureApis:
-    otm: OtmV1
+    otm_v1: OtmV1
+    otm_v2: OtmV2
```

Comment:
When will these be promoted to non-future APIs, out of interest?

Reply:
Unaware of an official timeline but I assume at SCO or post-SCO when customer contracts in Salesforce are in full swing